eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/JournalHead.scala

```diff
@@ -15,13 +15,19 @@ import com.evolutiongaming.scassandra.{DecodeRow, EncodeRow}
   * @param segmentSize the segmentSize used to write a journal to Cassandra
   * @param seqNr the sequence number of the last event in journal stored in Cassandra
   * @param deleteTo the last sequence number deleted from journal if any
+  * @param expiry the expiry time of the journal. On expiry the journal will be purged
+  * @param recordId unique ID of metadata entry, created once and _never_ updated. 
```

Comment:
I'd move the bulk of description to scaladoc of `RecordId` (check my comment there). Here it's better to describe in which cases `recordId` can be None - when we will look at the code 2 years from now, it might appear as if the field was always there and one might wonder what's the special meaning behind None case here.

Reply:
I added explanation _why_ its optional and _when_ we can make it required field